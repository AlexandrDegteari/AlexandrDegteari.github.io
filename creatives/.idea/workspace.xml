<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="ChangeListManager">
    <list default="true" id="b0d99af6-3a32-4c7a-b889-6191d56dfc79" name="Default" comment="" />
    <option name="EXCLUDED_CONVERTED_TO_IGNORED" value="true" />
    <option name="TRACKING_ENABLED" value="true" />
    <option name="SHOW_DIALOG" value="false" />
    <option name="HIGHLIGHT_CONFLICTS" value="true" />
    <option name="HIGHLIGHT_NON_ACTIVE_CHANGELIST" value="false" />
    <option name="LAST_RESOLUTION" value="IGNORE" />
  </component>
  <component name="FileEditorManager">
    <splitter split-orientation="horizontal" split-proportion="0.64166665">
      <split-first>
        <leaf SIDE_TABS_SIZE_LIMIT_KEY="300">
          <file leaf-file-name="index.html" pinned="false" current-in-tab="true">
            <entry file="file://$PROJECT_DIR$/index.html">
              <provider selected="true" editor-type-id="text-editor">
                <state relative-caret-position="546">
                  <caret line="162" column="41" lean-forward="true" selection-start-line="162" selection-start-column="41" selection-end-line="162" selection-end-column="41" />
                  <folding>
                    <element signature="n#style#0;n#div#3;n#body#0;n#html#0;n#!!top" expanded="true" />
                  </folding>
                </state>
              </provider>
            </entry>
          </file>
        </leaf>
      </split-first>
      <split-second>
        <leaf SIDE_TABS_SIZE_LIMIT_KEY="300">
          <file leaf-file-name="style.css" pinned="false" current-in-tab="true">
            <entry file="file://$PROJECT_DIR$/ﾑ《s/style.css">
              <provider selected="true" editor-type-id="text-editor">
                <state relative-caret-position="447">
                  <caret line="429" selection-start-line="429" selection-end-line="429" />
                </state>
              </provider>
            </entry>
          </file>
        </leaf>
      </split-second>
    </splitter>
  </component>
  <component name="FileTemplateManagerImpl">
    <option name="RECENT_TEMPLATES">
      <list>
        <option value="HTML File" />
        <option value="CSS File" />
      </list>
    </option>
  </component>
  <component name="FindInProjectRecents">
    <findStrings>
      <find>blog</find>
    </findStrings>
  </component>
  <component name="IdeDocumentHistory">
    <option name="CHANGED_PATHS">
      <list>
        <option value="$PROJECT_DIR$/style.css" />
        <option value="$PROJECT_DIR$/fonts/Gotham-Book/styles.css" />
        <option value="$PROJECT_DIR$/Css/style.css" />
        <option value="$PROJECT_DIR$/index.html" />
        <option value="$PROJECT_DIR$/ﾑ《s/style.css" />
      </list>
    </option>
  </component>
  <component name="JsBuildToolGruntFileManager" detection-done="true" sorting="DEFINITION_ORDER" />
  <component name="JsBuildToolPackageJson" detection-done="true" sorting="DEFINITION_ORDER" />
  <component name="JsGulpfileManager">
    <detection-done>true</detection-done>
    <sorting>DEFINITION_ORDER</sorting>
  </component>
  <component name="NodePackageJsonFileManager">
    <packageJsonPaths />
  </component>
  <component name="ProjectFrameBounds" extendedState="6">
    <option name="x" value="105" />
    <option name="y" value="107" />
    <option name="width" value="1400" />
    <option name="height" value="1000" />
  </component>
  <component name="ProjectView">
    <navigator proportions="" version="1">
      <foldersAlwaysOnTop value="true" />
    </navigator>
    <panes>
      <pane id="ProjectPane">
        <subPane>
          <expand>
            <path>
              <item name="Creatives" type="b2602c69:ProjectViewProjectNode" />
              <item name="Creatives" type="462c0819:PsiDirectoryNode" />
            </path>
            <path>
              <item name="Creatives" type="b2602c69:ProjectViewProjectNode" />
              <item name="Creatives" type="462c0819:PsiDirectoryNode" />
              <item name="ﾑ《s" type="462c0819:PsiDirectoryNode" />
            </path>
          </expand>
          <select />
        </subPane>
      </pane>
      <pane id="Scope" />
    </panes>
  </component>
  <component name="PropertiesComponent">
    <property name="DefaultHtmlFileTemplate" value="HTML File" />
    <property name="WebServerToolWindowFactoryState" value="false" />
    <property name="list.type.of.created.stylesheet" value="CSS" />
    <property name="nodejs_interpreter_path.stuck_in_default_project" value="undefined stuck path" />
    <property name="nodejs_npm_path_reset_for_default_project" value="true" />
  </component>
  <component name="RecentsManager">
    <key name="CopyFile.RECENT_KEYS">
      <recent name="C:\Users\Alex\PhpstormProjects\portfolio" />
    </key>
  </component>
  <component name="RunDashboard">
    <option name="ruleStates">
      <list>
        <RuleState>
          <option name="name" value="ConfigurationTypeDashboardGroupingRule" />
        </RuleState>
        <RuleState>
          <option name="name" value="StatusDashboardGroupingRule" />
        </RuleState>
      </list>
    </option>
  </component>
  <component name="SvnConfiguration">
    <configuration />
  </component>
  <component name="TaskManager">
    <task active="true" id="Default" summary="Default task">
      <changelist id="b0d99af6-3a32-4c7a-b889-6191d56dfc79" name="Default" comment="" />
      <created>1537801151761</created>
      <option name="number" value="Default" />
      <option name="presentableId" value="Default" />
      <updated>1537801151761</updated>
      <workItem from="1537801153073" duration="7533000" />
      <workItem from="1537864012109" duration="34364000" />
      <workItem from="1537949855191" duration="1886000" />
      <workItem from="1538050407291" duration="3687000" />
      <workItem from="1538058558409" duration="13165000" />
      <workItem from="1538121511394" duration="9351000" />
      <workItem from="1538159566774" duration="458000" />
      <workItem from="1538163428517" duration="9309000" />
    </task>
    <servers />
  </component>
  <component name="TimeTrackingManager">
    <option name="totallyTimeSpent" value="79753000" />
  </component>
  <component name="ToolWindowManager">
    <frame x="-8" y="-8" width="1936" height="1066" extended-state="6" />
    <editor active="true" />
    <layout>
      <window_info active="true" content_ui="combo" id="Project" order="1" visible="true" weight="0.25" />
      <window_info anchor="bottom" id="TODO" order="11" />
      <window_info anchor="bottom" id="Docker" order="0" show_stripe_button="false" />
      <window_info anchor="bottom" id="Event Log" order="6" side_tool="true" weight="0.32912722" />
      <window_info anchor="right" id="Database" order="0" />
      <window_info anchor="bottom" id="Database Changes" order="1" show_stripe_button="false" />
      <window_info anchor="bottom" id="Version Control" order="5" show_stripe_button="false" />
      <window_info anchor="bottom" id="Run" order="7" />
      <window_info id="Structure" order="2" side_tool="true" weight="0.25" />
      <window_info anchor="bottom" id="Terminal" order="2" />
      <window_info id="Favorites" order="0" side_tool="true" />
      <window_info anchor="bottom" id="Debug" order="8" weight="0.4" />
      <window_info anchor="right" content_ui="combo" id="Hierarchy" order="3" weight="0.25" />
      <window_info anchor="bottom" id="Inspection" order="10" weight="0.4" />
      <window_info anchor="right" id="Commander" internal_type="SLIDING" order="1" type="SLIDING" weight="0.4" />
      <window_info anchor="right" id="Ant Build" order="2" weight="0.25" />
      <window_info anchor="bottom" id="Message" order="3" />
      <window_info anchor="bottom" id="Cvs" order="9" weight="0.25" />
      <window_info anchor="bottom" id="Find" order="4" />
    </layout>
  </component>
  <component name="TypeScriptGeneratedFilesManager">
    <option name="version" value="1" />
  </component>
  <component name="VcsContentAnnotationSettings">
    <option name="myLimit" value="2678400000" />
  </component>
  <component name="editorHistoryManager">
    <entry file="file://$PROJECT_DIR$/index.html">
      <provider selected="true" editor-type-id="text-editor">
        <state relative-caret-position="289">
          <caret line="17" column="43" selection-start-line="17" selection-start-column="43" selection-end-line="17" selection-end-column="43" />
          <folding>
            <element signature="n#style#0;n#div#3;n#body#0;n#html#0;n#!!top" expanded="true" />
          </folding>
        </state>
      </provider>
    </entry>
    <entry file="file://$PROJECT_DIR$/ﾑ《s/style.css">
      <provider selected="true" editor-type-id="text-editor">
        <state relative-caret-position="527">
          <caret line="31" column="8" lean-forward="true" selection-start-line="31" selection-start-column="8" selection-end-line="31" selection-end-column="8" />
        </state>
      </provider>
    </entry>
    <entry file="file://$PROJECT_DIR$/index.html">
      <provider selected="true" editor-type-id="text-editor">
        <state relative-caret-position="1751">
          <caret line="103" column="33" selection-start-line="103" selection-start-column="33" selection-end-line="103" selection-end-column="33" />
        </state>
      </provider>
    </entry>
    <entry file="file://$PROJECT_DIR$/Css/style.css" />
    <entry file="file://$PROJECT_DIR$/index.html">
      <provider selected="true" editor-type-id="text-editor">
        <state relative-caret-position="5610">
          <caret line="330" column="17" lean-forward="true" selection-start-line="330" selection-start-column="17" selection-end-line="330" selection-end-column="17" />
        </state>
      </provider>
    </entry>
    <entry file="file://$PROJECT_DIR$/Css/style.css" />
    <entry file="file://$PROJECT_DIR$/index.html">
      <provider selected="true" editor-type-id="text-editor">
        <state relative-caret-position="357">
          <caret line="21" column="33" lean-forward="true" selection-start-line="21" selection-start-column="33" selection-end-line="21" selection-end-column="33" />
        </state>
      </provider>
    </entry>
    <entry file="file://$PROJECT_DIR$/Css/style.css" />
    <entry file="file://$PROJECT_DIR$/index.html">
      <provider selected="true" editor-type-id="text-editor">
        <state relative-caret-position="6783">
          <caret line="399" column="39" selection-start-line="399" selection-start-column="39" selection-end-line="399" selection-end-column="39" />
        </state>
      </provider>
    </entry>
    <entry file="file://$PROJECT_DIR$/Css/style.css" />
    <entry file="file://$PROJECT_DIR$/index.html">
      <provider selected="true" editor-type-id="text-editor">
        <state relative-caret-position="714">
          <caret line="42" column="32" lean-forward="true" selection-start-line="42" selection-start-column="32" selection-end-line="42" selection-end-column="32" />
        </state>
      </provider>
    </entry>
    <entry file="file://$PROJECT_DIR$/Css/style.css" />
    <entry file="file://$PROJECT_DIR$/style.css" />
    <entry file="file://$PROJECT_DIR$/fonts/Gotham-Book/styles.css" />
    <entry file="file://$PROJECT_DIR$/../Creatives2/css/font-awesome.css">
      <provider selected="true" editor-type-id="text-editor">
        <state relative-caret-position="238">
          <caret line="14" column="24" lean-forward="true" selection-start-line="14" selection-start-column="24" selection-end-line="14" selection-end-column="24" />
        </state>
      </provider>
    </entry>
    <entry file="file://$PROJECT_DIR$/Css/style.css" />
    <entry file="file://$PROJECT_DIR$/../Creatives2/css/style.css">
      <provider selected="true" editor-type-id="text-editor" />
    </entry>
    <entry file="file://$PROJECT_DIR$/../Portfolio/portfolio.html" />
    <entry file="file://$PROJECT_DIR$/index.html">
      <provider selected="true" editor-type-id="text-editor">
        <state relative-caret-position="546">
          <caret line="162" column="41" lean-forward="true" selection-start-line="162" selection-start-column="41" selection-end-line="162" selection-end-column="41" />
          <folding>
            <element signature="n#style#0;n#div#3;n#body#0;n#html#0;n#!!top" expanded="true" />
          </folding>
        </state>
      </provider>
    </entry>
    <entry file="file://$PROJECT_DIR$/ﾑ《s/style.css">
      <provider selected="true" editor-type-id="text-editor">
        <state relative-caret-position="447">
          <caret line="429" selection-start-line="429" selection-end-line="429" />
        </state>
      </provider>
    </entry>
  </component>
</project>