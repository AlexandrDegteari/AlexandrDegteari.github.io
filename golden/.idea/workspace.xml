<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="ChangeListManager">
    <list default="true" id="8a6961e9-1cab-4a33-b16c-984d01139ea7" name="Default" comment="" />
    <option name="EXCLUDED_CONVERTED_TO_IGNORED" value="true" />
    <option name="TRACKING_ENABLED" value="true" />
    <option name="SHOW_DIALOG" value="false" />
    <option name="HIGHLIGHT_CONFLICTS" value="true" />
    <option name="HIGHLIGHT_NON_ACTIVE_CHANGELIST" value="false" />
    <option name="LAST_RESOLUTION" value="IGNORE" />
  </component>
  <component name="FileEditorManager">
    <splitter split-orientation="horizontal" split-proportion="0.5">
      <split-first>
        <leaf SIDE_TABS_SIZE_LIMIT_KEY="300">
          <file leaf-file-name="index.html" pinned="false" current-in-tab="true">
            <entry file="file://$PROJECT_DIR$/index.html">
              <provider selected="true" editor-type-id="text-editor">
                <state relative-caret-position="1025">
                  <caret line="149" column="41" lean-forward="true" selection-start-line="149" selection-start-column="41" selection-end-line="149" selection-end-column="41" />
                </state>
              </provider>
            </entry>
          </file>
        </leaf>
      </split-first>
      <split-second>
        <leaf SIDE_TABS_SIZE_LIMIT_KEY="300">
          <file leaf-file-name="main.css" pinned="false" current-in-tab="true">
            <entry file="file://$PROJECT_DIR$/css/main.css">
              <provider selected="true" editor-type-id="text-editor">
                <state relative-caret-position="597">
                  <caret line="236" column="18" lean-forward="true" selection-start-line="236" selection-start-column="18" selection-end-line="236" selection-end-column="18" />
                </state>
              </provider>
            </entry>
          </file>
        </leaf>
      </split-second>
    </splitter>
  </component>
  <component name="FileTemplateManagerImpl">
    <option name="RECENT_TEMPLATES">
      <list>
        <option value="HTML File" />
        <option value="CSS File" />
      </list>
    </option>
  </component>
  <component name="IdeDocumentHistory">
    <option name="CHANGED_PATHS">
      <list>
        <option value="$PROJECT_DIR$/style.css" />
        <option value="$PROJECT_DIR$/css/main.css" />
        <option value="$PROJECT_DIR$/index.html" />
      </list>
    </option>
  </component>
  <component name="JsBuildToolGruntFileManager" detection-done="true" sorting="DEFINITION_ORDER" />
  <component name="JsBuildToolPackageJson" detection-done="true" sorting="DEFINITION_ORDER" />
  <component name="JsGulpfileManager">
    <detection-done>true</detection-done>
    <sorting>DEFINITION_ORDER</sorting>
  </component>
  <component name="NodePackageJsonFileManager">
    <packageJsonPaths />
  </component>
  <component name="ProjectFrameBounds" extendedState="6">
    <option name="x" value="105" />
    <option name="y" value="107" />
    <option name="width" value="1400" />
    <option name="height" value="1000" />
  </component>
  <component name="ProjectView">
    <navigator proportions="" version="1">
      <foldersAlwaysOnTop value="true" />
    </navigator>
    <panes>
      <pane id="ProjectPane">
        <subPane>
          <expand>
            <path>
              <item name="golden" type="b2602c69:ProjectViewProjectNode" />
              <item name="golden" type="462c0819:PsiDirectoryNode" />
            </path>
            <path>
              <item name="golden" type="b2602c69:ProjectViewProjectNode" />
              <item name="golden" type="462c0819:PsiDirectoryNode" />
              <item name="css" type="462c0819:PsiDirectoryNode" />
            </path>
          </expand>
          <select />
        </subPane>
      </pane>
      <pane id="Scope" />
    </panes>
  </component>
  <component name="PropertiesComponent">
    <property name="DefaultHtmlFileTemplate" value="HTML File" />
    <property name="WebServerToolWindowFactoryState" value="false" />
    <property name="list.type.of.created.stylesheet" value="CSS" />
    <property name="nodejs_interpreter_path.stuck_in_default_project" value="undefined stuck path" />
    <property name="nodejs_npm_path_reset_for_default_project" value="true" />
  </component>
  <component name="RunDashboard">
    <option name="ruleStates">
      <list>
        <RuleState>
          <option name="name" value="ConfigurationTypeDashboardGroupingRule" />
        </RuleState>
        <RuleState>
          <option name="name" value="StatusDashboardGroupingRule" />
        </RuleState>
      </list>
    </option>
  </component>
  <component name="SvnConfiguration">
    <configuration />
  </component>
  <component name="TaskManager">
    <task active="true" id="Default" summary="Default task">
      <changelist id="8a6961e9-1cab-4a33-b16c-984d01139ea7" name="Default" comment="" />
      <created>1537951821210</created>
      <option name="number" value="Default" />
      <option name="presentableId" value="Default" />
      <updated>1537951821210</updated>
      <workItem from="1537951823146" duration="15410000" />
      <workItem from="1537974077098" duration="741000" />
      <workItem from="1537992161708" duration="764000" />
      <workItem from="1538039020019" duration="2338000" />
      <workItem from="1538057411132" duration="999000" />
      <workItem from="1538060255514" duration="360000" />
    </task>
    <servers />
  </component>
  <component name="TimeTrackingManager">
    <option name="totallyTimeSpent" value="20612000" />
  </component>
  <component name="ToolWindowManager">
    <frame x="-8" y="-8" width="1936" height="1066" extended-state="6" />
    <editor active="true" />
    <layout>
      <window_info active="true" content_ui="combo" id="Project" order="0" visible="true" weight="0.18020834" />
      <window_info anchor="bottom" id="TODO" order="6" />
      <window_info anchor="bottom" id="Docker" order="7" show_stripe_button="false" />
      <window_info anchor="bottom" id="Event Log" order="7" side_tool="true" />
      <window_info anchor="right" id="Database" order="3" />
      <window_info anchor="bottom" id="Database Changes" order="7" show_stripe_button="false" />
      <window_info anchor="bottom" id="Run" order="2" />
      <window_info anchor="bottom" id="Version Control" order="7" show_stripe_button="false" />
      <window_info id="Structure" order="1" side_tool="true" weight="0.25" />
      <window_info anchor="bottom" id="Terminal" order="7" />
      <window_info anchor="bottom" id="Debug" order="3" weight="0.4" />
      <window_info id="Favorites" order="2" side_tool="true" />
      <window_info anchor="right" content_ui="combo" id="Hierarchy" order="2" weight="0.25" />
      <window_info anchor="bottom" id="Inspection" order="5" weight="0.4" />
      <window_info anchor="right" id="Commander" internal_type="SLIDING" order="0" type="SLIDING" weight="0.4" />
      <window_info anchor="right" id="Ant Build" order="1" weight="0.25" />
      <window_info anchor="bottom" id="Message" order="0" />
      <window_info anchor="bottom" id="Cvs" order="4" weight="0.25" />
      <window_info anchor="bottom" id="Find" order="1" />
    </layout>
  </component>
  <component name="TypeScriptGeneratedFilesManager">
    <option name="version" value="1" />
  </component>
  <component name="VcsContentAnnotationSettings">
    <option name="myLimit" value="2678400000" />
  </component>
  <component name="editorHistoryManager">
    <entry file="file://$PROJECT_DIR$/index.html">
      <provider selected="true" editor-type-id="text-editor">
        <state relative-caret-position="238">
          <caret line="14" column="17" lean-forward="true" selection-start-line="14" selection-start-column="17" selection-end-line="14" selection-end-column="17" />
        </state>
      </provider>
    </entry>
    <entry file="file://$PROJECT_DIR$/css/main.css">
      <provider selected="true" editor-type-id="text-editor">
        <state relative-caret-position="714">
          <caret line="42" column="19" lean-forward="true" selection-start-line="42" selection-start-column="19" selection-end-line="42" selection-end-column="19" />
        </state>
      </provider>
    </entry>
    <entry file="file://$PROJECT_DIR$/index.html">
      <provider selected="true" editor-type-id="text-editor">
        <state relative-caret-position="2312">
          <caret line="136" column="35" lean-forward="true" selection-start-line="136" selection-start-column="35" selection-end-line="136" selection-end-column="35" />
        </state>
      </provider>
    </entry>
    <entry file="file://$PROJECT_DIR$/css/main.css">
      <provider selected="true" editor-type-id="text-editor">
        <state relative-caret-position="3519">
          <caret line="207" column="19" lean-forward="true" selection-start-line="207" selection-start-column="19" selection-end-line="207" selection-end-column="19" />
        </state>
      </provider>
    </entry>
    <entry file="file://$PROJECT_DIR$/index.html">
      <provider selected="true" editor-type-id="text-editor">
        <state relative-caret-position="2567">
          <caret line="151" column="27" lean-forward="true" selection-start-line="151" selection-start-column="27" selection-end-line="151" selection-end-column="27" />
        </state>
      </provider>
    </entry>
    <entry file="file://$PROJECT_DIR$/css/main.css">
      <provider selected="true" editor-type-id="text-editor">
        <state relative-caret-position="1377">
          <caret line="81" selection-start-line="81" selection-end-line="81" />
        </state>
      </provider>
    </entry>
    <entry file="file://$PROJECT_DIR$/index.html">
      <provider selected="true" editor-type-id="text-editor">
        <state relative-caret-position="2329">
          <caret line="137" column="50" selection-start-line="137" selection-start-column="50" selection-end-line="137" selection-end-column="50" />
        </state>
      </provider>
    </entry>
    <entry file="file://$PROJECT_DIR$/css/main.css">
      <provider selected="true" editor-type-id="text-editor">
        <state relative-caret-position="1377">
          <caret line="81" selection-start-line="81" selection-end-line="81" />
        </state>
      </provider>
    </entry>
    <entry file="file://$PROJECT_DIR$/index.html">
      <provider selected="true" editor-type-id="text-editor">
        <state relative-caret-position="2329">
          <caret line="137" column="50" selection-start-line="137" selection-start-column="50" selection-end-line="137" selection-end-column="50" />
        </state>
      </provider>
    </entry>
    <entry file="file://$PROJECT_DIR$/css/main.css">
      <provider selected="true" editor-type-id="text-editor">
        <state relative-caret-position="3519">
          <caret line="207" column="23" lean-forward="true" selection-start-line="207" selection-start-column="23" selection-end-line="207" selection-end-column="23" />
        </state>
      </provider>
    </entry>
    <entry file="file://$PROJECT_DIR$/style.css" />
    <entry file="file://$PROJECT_DIR$/css/main.css">
      <provider selected="true" editor-type-id="text-editor">
        <state relative-caret-position="597">
          <caret line="236" column="18" lean-forward="true" selection-start-line="236" selection-start-column="18" selection-end-line="236" selection-end-column="18" />
        </state>
      </provider>
    </entry>
    <entry file="file://$PROJECT_DIR$/index.html">
      <provider selected="true" editor-type-id="text-editor">
        <state relative-caret-position="1025">
          <caret line="149" column="41" lean-forward="true" selection-start-line="149" selection-start-column="41" selection-end-line="149" selection-end-column="41" />
        </state>
      </provider>
    </entry>
  </component>
</project>